$OpenBSD: patch-configure_ac,v 1.1 2011/10/16 21:24:04 jasper Exp $

- Don't check for gnome-doc-utils, there are no docs to build anyway as they
  have been removed some time ago.

--- configure.ac.orig	Sun Oct 16 23:18:58 2011
+++ configure.ac	Sun Oct 16 23:19:54 2011
@@ -18,7 +18,6 @@ LT_INIT
 IT_PROG_INTLTOOL([0.35.0])
 
 GNOME_COMPILE_WARNINGS
-GNOME_DOC_INIT
 
 AC_PATH_PROG(GLIB_GENMARSHAL, glib-genmarshal)
 
@@ -56,14 +55,14 @@ if test "$GCC" = "yes"; then
 	WARNINGFLAGS="$WARNINGFLAGS -Wformat-security"
 	WARNINGFLAGS="$WARNINGFLAGS -Wmissing-include-dirs"
 	WARNINGFLAGS="$WARNINGFLAGS -Wmissing-format-attribute"
-	WARNINGFLAGS="$WARNINGFLAGS -Wclobbered"
-	WARNINGFLAGS="$WARNINGFLAGS -Wempty-body"
-	WARNINGFLAGS="$WARNINGFLAGS -Wignored-qualifiers"
-	WARNINGFLAGS="$WARNINGFLAGS -Wtype-limits"
+#	WARNINGFLAGS="$WARNINGFLAGS -Wclobbered"
+#	WARNINGFLAGS="$WARNINGFLAGS -Wempty-body"
+#	WARNINGFLAGS="$WARNINGFLAGS -Wignored-qualifiers"
+#	WARNINGFLAGS="$WARNINGFLAGS -Wtype-limits"
 	WARNINGFLAGS="$WARNINGFLAGS -Wdeclaration-after-statement"
 	WARNINGFLAGS="$WARNINGFLAGS -Wshadow"
 	WARNINGFLAGS="$WARNINGFLAGS -Winline"
-	WARNINGFLAGS="$WARNINGFLAGS -Wmissing-parameter-type"
+#	WARNINGFLAGS="$WARNINGFLAGS -Wmissing-parameter-type"
 	WARNINGFLAGS="$WARNINGFLAGS -Woverride-init"
 	WARNINGFLAGS="$WARNINGFLAGS -Wno-strict-aliasing"
 else
